src/database/lib/threadDB.ts

```diff
@@ -0,0 +1,248 @@
+const { Where } = require('@textile/hub');
```

Comment:
threadDb related calls should be present in `utils` and `registerNewUser`, `getLoginUser` or anything related to Safient workflow should be moved to `logic`